circus-train-aws-sns/src/test/java/com/hotels/bdp/circustrain/aws/sns/event/SnsListenerTest.java

```diff
@@ -115,20 +132,30 @@ public void start() {
     PublishRequest request = requestCaptor.getValue();
     assertThat(request.getSubject(), is(SUBJECT));
     assertThat(request.getTopicArn(), is("startArn"));
-    assertThat(request.getMessage(),
-        is("{\"protocolVersion\":\"1.0\",\"type\":\"START\",\"headers\":{\"pipeline-id\":\"0943879438\"},"
-            + "\"startTime\":\"starttime\",\"eventId\":\"EVENT_ID\",\"sourceCatalog\":\"sourceCatalogName\","
-            + "\"replicaCatalog\":\"replicaCatalogName\",\"sourceTable\":\"srcDb.srcTable\",\"replicaTable\":"
-            + "\"replicaDb.replicaTable\"}"));
+    assertThat(request.getMessage(), is("{"
+        + PROTOCOL_VERSION
+        + ",\"type\":\"START\",\"headers\":{\"pipeline-id\":\"0943879438\"},"
+        + "\"startTime\":\"starttime\",\"eventId\":\"EVENT_ID\",\"sourceCatalog\":\"sourceCatalogName\","
+        + "\"replicaCatalog\":\"replicaCatalogName\",\"sourceTable\":\"srcDb.srcTable\",\"replicaTable\":"
+        + "\"replicaDb.replicaTable\",\"replicaTableLocation\":\"s3://bucket/path\",\"replicaMetastoreUris\":\"thrift://host:9083\"}"));
   }
 
+  // TODO: need tests for unpartitioned tables to see what gets output for partition stuff
```

Comment:
still going todo?

Reply:
No, it's done in successUnpartitionedTable(), will remove the todo, thanks for spotting.